isis/src/base/apps/phocube/main.cpp

```diff
@@ -447,15 +441,35 @@ void phocube(Buffer &out) {
             index += 64 * 64;
           }
         }
-        
       }
-      // Trim outerspace
+
+      // Trim outer space except last two bands when RA and declination are selected
+      // We always skip two because RA and declination are selected together.
       else {
-        for(int b = (skipDN) ? 1 : 0; b < nbands; b++) {
+        int bandsToProcess;
+        if (ra && declination) {
+          bandsToProcess = nbands - 2;
+        }
+        else {
+          bandsToProcess = nbands;
+        }
+
+        for (int b = (skipDN) ? 1 : 0; b < bandsToProcess; b++) {
           out[index] = Isis::NULL8;
           index += 64 * 64;
         }
       }
+
+      // Export the RA and declination regardless if the point is in outer space or not.
+      if (ra) {
```

Comment:
I wasn't sure about having separate if statements and associated variables `ra` and `declination` initially since ra and dec are set from the same `RADEC` user parameters, but it is consistent with the way BodyFixedX, BodyFixedY, BodyFixedZ are handled in this application, so :+1:!